### PR TITLE
chore: update organization name wherever possible

### PR DESCRIPTION
**Pull request by Readpato**
### 🔗 Linked issue

Addresses #134

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Utilizes "schroedinger" instead of "schrodinger" wherever possible.

We still include allow some "schrodinger" in the repository given that there are some places where we cannot implement it yet (Twitter, Linkedin, etc)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -31,8 +31,7 @@ body:
       label: Final checks
       description: Before submitting, please make sure you do the following
       options:
-        - label: Read the [code of conduct](https://www.schrodinger-hat.it/code-of-conduct).
+        - label: Read the [code of conduct](https://www.schroedinger-hat.org/code-of-conduct).
           required: true
-        - label: Check existing [issues](https://github.com/Schrodinger-Hat/schrodinger-hat-website/issues).
+        - label: Check existing [issues](https://github.com/schroedinger-Hat/schrodinger-hat-website/issues).
```
Comment (Wabri) [inline .github/ISSUE_TEMPLATE/feature_request.yml:36]: Maybe I will wait the decision on this #136

**Review by Wabri — COMMENTED** [review]
Lets wait for #136

**Review by TheJoin95 — APPROVED** [review]

**Comment by TheJoin95** [comment]
note: we should do a grep search on the whole schoedinger-hat organisation to update anywhere

**Comment by Readpato** [comment]
Should we merge it directly or should we wait for https://github.com/Schroedinger-Hat/schroedinger-hat/issues/227?

I think it will be best to merge it right away to then tackle the social media links, cc @TheJoin95 @Wabri

**Comment by TheJoin95** [comment]
To me this can be merged. To double check with Gabri who is the assignee of that issue

**Comment by Readpato** [comment]
Given me missing the information that you cant update the Twitter, LinkedIn and Instagram handle (#192) I will proceed with the merging of this PR

File: src/pages/PageHome.vue [hunk 2]
```diff
@@ -12,7 +12,7 @@ const links = [
     text: 'OSDay 2024',
   },
   {
-    href: 'https://opencollective.com/schrodinger-hat',
+    href: 'https://opencollective.com/schroedinger-hat',
```
Comment (Wabri) [inline src/pages/PageHome.vue:15]: We can't update the opencollective url

```suggestion
    href: 'https://opencollective.com/schrodinger-hat',
```

**Review by Wabri — CHANGES_REQUESTED** [review]

**Comment by Readpato** [comment]
@Wabri Please check PR #137 which addresses all your subsequent comments from this PR

**Pull request merged**